dags/bqetl_campaign_cost_breakdowns.py

```diff
@@ -0,0 +1,67 @@
+# Generated via https://github.com/mozilla/bigquery-etl/blob/main/bigquery_etl/query_scheduling/generate_airflow_dags.py
+
+from airflow import DAG
+from airflow.sensors.external_task import ExternalTaskMarker
+from airflow.sensors.external_task import ExternalTaskSensor
+from airflow.utils.task_group import TaskGroup
+import datetime
+from utils.constants import ALLOWED_STATES, FAILED_STATES
+from utils.gcp import bigquery_etl_query, gke_command
+
+docs = """
+### bqetl_campaign_cost_breakdowns
+
+Built from bigquery-etl repo, [`dags/bqetl_campaign_cost_breakdowns.py`](https://github.com/mozilla/bigquery-etl/blob/main/dags/bqetl_campaign_cost_breakdowns.py)
+
+#### Description
+
+Derived tables on top of fenix installation and DOU metrics, as well as Google ads campaign data.
+
+#### Owner
+
+ctroy@mozilla.com
+"""
+
+
+default_args = {
+    "owner": "ctroy@mozilla.com",
+    "start_date": datetime.datetime(2021, 9, 20, 0, 0),
+    "end_date": None,
+    "email": [
+        "ctroy@mozilla.com",
+        "wstuckey@mozilla.com",
+        "rburwei@mozilla.com",
+        "telemetry-alerts@mozilla.com",
+    ],
+    "depends_on_past": False,
+    "retry_delay": datetime.timedelta(seconds=1800),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+}
+
+tags = ["impact/tier_1", "repo/bigquery-etl"]
```

Comment:
What is impact/tier_1? Is that the appropriate tier for something like this?

Reply:
These are the [tier classifications](https://mozilla.github.io/bigquery-etl/reference/airflow_tags/). For the appropriate I'd ask: If it fails, how big is the impact?